main/src/com/google/refine/util/HttpClient.java

```diff
@@ -105,6 +111,24 @@ public void process(
                     }
                 });
 
+        if (System.getProperty("http.proxyHost") != null)
+            proxyHost = System.getProperty("http.proxyHost");
+
```

Comment:
I think we generally try to avoid if blocks without `{` and `}` because some consider it bad practice. This is something that should be caught by the linter, it looks like we need to configure it better. In the meantime, could you add the braces to those `if` blocks?